src/lib/parquet/tests/Flow/Parquet/Tests/Integration/IO/PaginationTest.php

```diff
@@ -13,6 +13,25 @@
 
 final class PaginationTest extends TestCase
 {
+    public function test_reading_last_100_rows() : void
+    {
+        $path = __DIR__ . '/Fixtures/pagination_row_group_1kb_5k_rows.snappy.parquet';
```

Comment:
Maybe it's a good idea to use a data provider here, because I see you are duplicating this value